### DISCUSSION TITLE
Kafka的 topic生成为题

### DISCUSSION THREAD

**Discussion by ownbylichaobao** [milvus-io/milvus#17110] [Q&A and General discussion] [answered]
请问有人知道Milvus里面Kafka的topic的生成逻辑么？以及Kafka的 Topic是固定的么，还是动态生成的？

**Comment by xiaofan-luan** [comment, marked as answer]
> 请问有人知道Milvus里面Kafka的topic的生成逻辑么？以及Kafka的 Topic是固定的么，还是动态生成的？

Topic 是集群启动时就生成好，正常就200多个吧，可以调小一些

**Reply by ownbylichaobao** [reply]
感谢老板
